espnet/nets/pytorch_backend/lm/ngram.py

```diff
@@ -0,0 +1,41 @@
+import kenlm
+import torch
+
+from espnet.nets.scorer_interface import ScorerInterface
+from espnet.nets.scorer_interface import PartialScorerInterface
+
+from abc import ABC
+
+
+class Ngrambase(ABC):
```

Comment:
Can you add a small test for this class `test/test_ngram.py`? Maybe it can be just checking the output scores with toy inputs and arpa lm.